### ISSUE TITLE
TestStoreRangeDownReplicate hangs

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#2547]
The following test appears to have failed:

[#7565](https://circleci.com/gh/cockroachdb/cockroach/7565):

```
I0917 19:40:05.068374 949 storage/range_gc_queue.go:124  destroying local data from range 2
W0917 19:40:05.071212 949 rpc/server.go:436  rpc: write response failed: write tcp 127.0.0.1:55807->127.0.0.1:60484: use of closed network connection
W0917 19:40:05.071371 949 rpc/server.go:436  rpc: write response failed: write tcp 127.0.0.1:55807->127.0.0.1:60484: use of closed network connection
W0917 19:40:05.073307 949 multiraft/multiraft.go:958  node 300000003 failed to send message to 100000001: multiraft/transport.go:135: transport is closed
W0917 19:40:05.073452 949 multiraft/multiraft.go:958  node 300000003 failed to send message to 100000001: multiraft/transport.go:135: transport is closed
--- FAIL: TestStoreRangeDownReplicate (62.85s)
	client_raft_test.go:681: Failed to achieve proper replication within 10 seconds
=== RUN   TestChangeReplicasDescriptorInvariant
I0917 19:40:05.080670 949 storage/replica.go:1234  gossiping cluster id  from store 1, range 1
I0917 19:40:05.081343 949 multiraft/multiraft.go:446  node 100000001 starting
I0917 19:40:05.081802 949 raft/raft.go:406  group 1 100000001 became follower at term 5
I0917 19:40:05.082516 949 raft/raft.go:219  group 1 newRaft 100000001 [peers: [100000001], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0917 19:40:05.082672 949 raft/raft.go:485  group 1 100000001 is starting a new election at term 5
I0917 19:40:05.082760 949 raft/raft.go:419  group 1 100000001 became candidate at term 6
I0917 19:40:05.082824 949 raft/raft.go:468  group 1 100000001 received vote from 100000001 at term 6
I0917 19:40:05.082922 949 raft/raft.go:447  group 1 100000001 became leader at term 6
--
c518820376611819.55          19:40:20.376784  1.370155ms  ·sending RPC             kv/dist_sender.go:424

--- PASS: TestStoreRangeSystemSplits (0.78s)
=== RUN   Example_rebalancing
--- PASS: Example_rebalancing (0.62s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	84.121s
=== RUN   TestBatchBasics
I0917 19:38:59.618568 961 storage/engine/rocksdb.go:122  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.01s)
=== RUN   TestBatchGet
I0917 19:38:59.623106 961 storage/engine/rocksdb.go:122  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.01s)
=== RUN   TestBatchMerge
I0917 19:38:59.629309 961 storage/engine/rocksdb.go:122  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

**Comment by mrtracy** [comment]
I've temporarily disabled the test in question; it might just be running slow, but the output also looks like it might be getting stuck.

**Comment by tschottdorf** [comment]
closing in favor of #2091

**Comment by mrtracy** [comment]
That other issue doesn't appear to be for the same test, and I'm about 50% sure this one isn't a fluke.

**Comment by tschottdorf** [comment]
sorry, I mistyped something here. Thanks for catching it.

**Comment by bdarnell** [comment]
Looks like this has been fixed; the offending test was re-enabled in #2755
